### PR TITLE
Remove broken links

### PR DESCRIPTION
**Pull request by freearhey**
All links are tested with `scripts/test.js` and the VLC player before removal.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Maybe consider fusioning this PR with mine ? Since we're almost doing the same things (counting namefixes and stream additions overtime)

**Comment by LaneSh4d0w** [comment]
IMO it looks good since it won't affect anything other than keeping all online channels. LGTM.

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -41,8 +35,6 @@ https://unlimited2-cl.dps.live/nettv/nettv.smil/playlist.m3u8
 http://coninfo.net:1935/13maxhd/live_720p/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/4/4e/Canal_10_Mar_del_Plata_%28Logo_2010%29.png" group-title="",Canal 10 Mar del Plata
 https://cdn2.zencast.tv:30443/live/canal10smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://pbs.twimg.com/profile_images/764094244722184192/ZzC-pGTJ.jpg" group-title="",Litus Tv
-http://192.99.38.174:1935/litustv/ngrp:litustv_all/playlist.m3u8
```
Comment (themak1985) [inline channels/unsorted.m3u:45]: Working Link

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -226,14 +193,10 @@ http://190.103.183.24:1935/live/MicroHD/chunklist.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://3.bp.blogspot.com/-1e0viy-XtK8/XYugidQtXhI/AAAAAAAAoBI/yv7hogt4R3EIPJHQ5zs7WI5PRSMrGV2rACLcBGAsYHQ/s200/Cotubanama.jpg" group-title="",CotubanamaTV (RD)
-https://cdn4.hostlagarto.com:8081/cotubanamatv/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://3.bp.blogspot.com/-1P8ozDYekaw/XbBkLrf0Q4I/AAAAAAAAodQ/5qTLYC9km6YVHW8E1WoELvFneAUm1U7bwCPcBGAYYCw/s200/Teleunion%2B%25281%2529.png" group-title="",Teleunion (RD)
 http://server3.prostudionetwork.com:1945/teleunion/TU/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://4.bp.blogspot.com/-V8ZyzsGFiOo/Wr0xO-i31hI/AAAAAAAAdG8/SrmsBscKPcI1KmcKvbhbVJA1gRXpw_vSwCLcBGAs/s200/Sabor%2Ba%2BMerengue.jpg" group-title="",Sabor a merengue (RD)
 http://ss6.domint.net:2004/sam_str/sabor-video/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://1.bp.blogspot.com/-5JBZ6ZO0ybk/XaS-xAYTF7I/AAAAAAAAoXA/u1OK-aVn_sQNM86uwWFBqLuk5Q-Uzo7hgCLcBGAsYHQ/s200/vegateve.jpg" group-title="",Vega Teve (RD)
-https://ss6.domint.net:3012/tes_str/teleelsalvador/playlist.m3u8
```
Comment (themak1985) [inline channels/unsorted.m3u:236]: Working Link

File: channels/us.m3u [hunk 3]
```diff
@@ -71,12 +69,6 @@ https://abclive1-lh.akamaihd.net/i/abc_live07@423401/master.m3u8
 https://abclive1-lh.akamaihd.net/i/abc_live08@423402/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/V8eTp2V.png" group-title="News",ABC News Digital 9
 https://abclive1-lh.akamaihd.net/i/abc_live09@420891/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/nEcuCl9.png" group-title="Local",Action News Jax (WJAX-TV)
-https://svc-lvanvato-cxtv-wjax.cmgvideo.com/wjax/2564k/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/MaQSYGw.jpg" group-title="",Action Sports 4U
-https://nimble.dashmedia.tv/onestudio/actionsportstv/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Entertainment",Adult Swim
-https://adultswimtv.warnermediacdn.com/hls/live/2007031/livestream/master.m3u8
```
Comment (themak1985) [inline channels/us.m3u:79]: working link

File: channels/us.m3u [hunk 4]
```diff
@@ -583,16 +561,12 @@ https://granicusliveus2-a.akamaihd.net/lfucg/G0264_002/playlist.m3u8
 https://uni5rtmp.tulix.tv/lifevision/lifevision.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/OaSFTnV.png" group-title="",Live Live
 https://dai.google.com/linear/hls/event/xC8SDBfbTKCTCa20kFJQXQ/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://lh3.googleusercontent.com/9AnBf9_s5RDDg6joBI3kPqglfVddV72Esw1jGBc4rGzlxUElacw4UQy_z2CTVnI7vHgZXDQbqp2Gw5UqzD3uvzkJXfT0YZbxKmoeNMo" group-title="",Local Now Philidelphia
-https://ottmanifest.weathergroup.com/live/10.233.185.101:58081/api/v1/live/hls/localnow/index.m3u8?network=OTT&experienceid=paPhiladelphia&seid=532865b2-796b-497d-abb2-56df16f62f32&showlocalnow=false&cdn=https%3a//ottcdn-a.akamaihd.net&sessionId=LocalNowWeb_bW9jLm9vYWNiQDM4NzkwZXd0&format=HLS&tpp=10443
```
Comment (themak1985) [inline channels/us.m3u:587]: Working Link

File: channels/us.m3u [hunk 5]
```diff
@@ -707,16 +679,10 @@ http://content.uplynk.com/channel/328d1434fb51476cb6567c74d5b2cc70.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/G37a3NG.png" group-title="Local",Oregon - Fox 12 News
-http://api.new.livestream.com/accounts/6908462/events/2706967/live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/9uz9lyg.png" group-title="",OUR TV
 https://hls-cdn.tvstartup.net/barakyah-channel/play/mp4:ourtvedge/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/BYNg7Wk.png" group-title="",Outdoor America
 https://dai.google.com/linear/hls/event/WB-7LjdsRVm0wVoLZjR8mA/master.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Urdu" tvg-logo="" group-title="",PakUs TV
-https://livestream.5centscdn.com/pakustv/d0dbe915091d400bd8ee7f27f0791303.sdp/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="" group-title="",PBC Tapesh TV
-https://iptv.tapesh.tv/tapesh/playlist.m3u8
```
Comment (themak1985) [inline channels/us.m3u:719]: Working Link

**Review by themak1985 — COMMENTED** [review]
I found 24 working links. The rest looks to be broken to me too.  @freearhey

File: channels/de.m3u [hunk 6]
```diff
@@ -205,10 +199,6 @@ http://rnf.iptv-playoutcenter.de:1935/rnf/rnf.stream_1/playlist.m3u8
 https://586fb512206e4.streamlock.net/rheinmaintv-live/_definst_/mp4:livestream/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="German" tvg-logo="https://i.imgur.com/YkB39hx.png" group-title="",RNF
 https://rnf.iptv-playoutcenter.de/rnf/rnf.stream_1/playlist.m3u8
-#EXTINF:-1 tvg-id="RTLNitro.de" tvg-name="RTL Nitro" tvg-language="German" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/RTL_Nitro_logo.svg/2000px-RTL_Nitro_logo.svg.png" group-title="",RTL Nitro
-http://188.35.9.5:10020/stolen/479d/hot.dll
-#EXTINF:-1 tvg-id="RTLPlus.de" tvg-name="RTL Plus" tvg-language="German" tvg-logo="https://cdn.static-fra.de/rtlplus/images/template/rtlplus_logo.png" group-title="",RTL Plus
-http://188.35.9.5:10020/stolen/480c/hot.dll
```
Comment (themak1985) [inline channels/de.m3u:219]: Working Link

**Comment by freearhey** [comment]
@themak1985 Thank you for your help! I've already returned all the links you marked.

**Pull request merged**